content/python/concepts/built-in-functions/terms/sum/sum.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: 'sum()'
+Description: 'Takes in an iterable object, such as a list or tuple, and returns the sum of all elements.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+  - 'Iterators'
+  - 'Lists'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+Takes in an iterable object, such as a list or tuple, and returns the sum of all elements.
+
+## Syntax
+
+```pseudo
+sum(iterable, start=0)
+```
+**iterable:** is an object capable of returning its elements one at a time
+for example a list, tuple or dictionary.
+
+**start:** is a number added to the sum of the numbers in the iterable.
+
+## Example 1
```

Comment:
Before the ` ```shell ` blocks can we add an introductory sentence such as:

```
This example results in the following output:
```

Reply:
@SSwiniarski I have added your changes and also the intro sentence for the  shell blocks